src/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -1684,4 +1684,22 @@
   <data name="CryptoError_Sha1" xml:space="preserve">
     <value>Unexpected cryptographic error while computing SHA1 ({0}).</value>
   </data>
+  <data name="ClassLoad_General" xml:space="preserve">
+    <value>Could not load type '{0}' from assembly '{1}'.</value>
+  </data>
+  <data name="ClassLoad_MissingMethodRva" xml:space="preserve">
+    <value>Could not load type '{0}' from assembly '{1}' because the method '{2}' has no implementation (no RVA).</value>
```

Comment:
(The compiler-writer aid that provides more detailed clues about BadImageFormat/InvalidProgram type of errors should be a separate tool, like PEVerify for desktop is today.)